### PR TITLE
Modified to use WebKitBrowser plugin from rdkservices instead of Thun…

### PR DESCRIPTION
**Pull request by neeraj-deshpande**
…derNanoServices.

### CODE REVIEW

File: recipes-wpe/wpeframework/wpeframework-rdkservices_git.bb [hunk 1]
```diff
@@ -31,7 +31,7 @@ PACKAGECONFIG ?= " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'playready_vg',         'opencdmi_vgrdm', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'widevine',             'opencdmi_wv', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'security',             'securityagent', '', d)} \
-    deviceinfo locationsync monitor tracing \
+    deviceinfo locationsync monitor tracing webkitbrowser apps ux \
 "
```
Comment (HaseenaSainul) [inline recipes-wpe/wpeframework/wpeframework-rdkservices_git.bb:35]: it is a cosmetic comments, could you please rearrange these packages in alphabetic order ?

**Review by HaseenaSainul — CHANGES_REQUESTED** [review]
This build will break AMLogic rdk build, since it is not able to provide a platform backend for DeviceIdentification plugin (due to the missing of FindMFRFWLibs.cmake in this sprint branch). Pull request is raised against rdkservices:sprint/2010 branch to add this support: [PR:623](`https://github.com/rdkcentral/rdkservices/pull/623`)

File: recipes-wpe/wpeframework/wpeframework-rdkservices_git.bb (hunk 1 above)
Comment (neeraj-deshpande) [reply recipes-wpe/wpeframework/wpeframework-rdkservices_git.bb:35]: Done.

**Review by HaseenaSainul — APPROVED** [review]
LGTM

**Pull request merged**